test/testutils/validators.go

```diff
@@ -373,3 +374,31 @@ func ValidateLatestEvent(ctx context.Context, k8sClient client.Client, eventReas
 
 	}, Timeout, Interval).Should(gomega.BeNil())
 }
+
+func ExpectWorkerStatefulSetsNotCreated(ctx context.Context, k8sClient client.Client, lws *leaderworkerset.LeaderWorkerSet) {
+	ginkgo.By("checking no worker statefulset created")
+	gomega.Consistently(func() bool {
+		var statefulSetList appsv1.StatefulSetList
+		if err := k8sClient.List(ctx, &statefulSetList, client.InNamespace(lws.Namespace), &client.MatchingLabels{leaderworkerset.SetNameLabelKey: lws.Name}); err != nil {
+			return false
+		}
+		return len(statefulSetList.Items) == 1 && statefulSetList.Items[0].Name == lws.Name
+	}, Timeout, Interval).Should(gomega.Equal(true))
+}
+
+func ExpectSpecifiedWorkerStatefulSetsCreated(ctx context.Context, k8sClient client.Client, workerStsName, namespace string) {
```

Comment:
Same there to unify the pattern:

```suggestion
func ExpectSpecifiedWorkerStatefulSetsCreated(ctx context.Context, k8sClient client.Client, lws *leaderworkerset.LeaderWorkerSet, start, end int32)
```